frontend/pkg/frontend/cluster.go

```diff
@@ -563,62 +583,67 @@ func (f *Frontend) updateHCPClusterInCosmos(ctx context.Context, writer http.Res
 		return err
 	}
 
+	// TODO is patch supposed to be status accepted?
```

Comment:
Wondered about this.  Nothing is failing, but should it be different?

Reply:
Yes, good catch.  Response code should be 202 (Accepted).

Resource Provider Contract quote (emphasis mine):
> **Response**
>
> The response includes an HTTP status code, a set of response headers, and a response body.
>
> **Status Code**
>
> The resource provider must return 200 (OK) to indicate that the synchronous Patch operation completed successfully. **202 (Accepted) must be returned to indicate that the operation will complete asynchronously.**
>
> If the resource group *or* resource does not exist, 404 (NotFound) should be returned.